oracle/config/affirmation-request-watcher.config.js

```diff
@@ -26,7 +26,7 @@ module.exports =
         ...baseConfig.bridgeConfig,
         ...baseConfig.foreignConfig,
         event: 'Transfer',
-        eventContractAddress: process.env.ERC20_TOKEN_ADDRESS,
+        eventContractAddress: initialChecks.ERC20_TOKEN_ADDRESS,
```

Comment:
let's keep consistency in the variables names: use the 'camel'-style (as `initialChecks.foreignERC` above) instead of `_`.

Also please consider to rename the variable to `bridgeableTokenAddress`.

Reply:
Done, renamed `ERC20_TOKEN_ADDRESS` to `bridgeableTokenAddress`, in other places in code as well.

Reply:
thanks